cosmoz-image-viewer.js

```diff
@@ -447,8 +455,9 @@
 		},
 
 		createZipFromUrls(fileUrls) {
+			const credentials = this.credentials ? { credentials: 'include' } : { credentials: 'omit' };
 			const fetches = fileUrls.map(url =>
-				fetch(url)
+				fetch(url, credentials)
```

Comment:
The second param should be `init` (like the docs) or `options` or something similar, since it could contain other properties in addition to credentials settings, for readability/clarity.

Reply:
Yes, agreed, options would be more clear as a name. Renaming.